play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -308,8 +326,24 @@ class OptionsPanel extends PSRoomPanel {
 					class="trainersprite yours" width="40" height="40" style={{ verticalAlign: 'middle' }}
 					src={Dex.resolveAvatar(`${PS.user.avatar}`)}
 				/> {}
-				{PS.user.name}
+				<strong> {PS.user.name} </strong>
```

Comment:
```suggestion
				<strong>{PS.user.name}</strong>
```

In general we prefer not to space our HTML tags.